c-abi doesn't work with 128bit flag

running:

cargo test --all --features c-abi,128bit

causes:

```
   Compiling value-trait v0.2.9-alpha.0 (/home/heinz/Projects/simd-lite/value-trait)
error[E0599]: the associated item `STABLE_ABI` exists for struct `GetTypeLayoutCtor<i128>`, but its trait bounds were not satisfied
  --> src/node.rs:15:38
   |
15 | #[cfg_attr(feature = "c-abi", derive(abi_stable::StableAbi))]
   |                                      ^^^^^^^^^^^^^^^^^^^^^ associated item cannot be called on `GetTypeLayoutCtor<i128>` due to unsatisfied trait bounds
   |
   = note: the following trait bounds were not satisfied:
           `i128: StableAbi`
   = note: this error originates in the macro `$crate::_sabi_type_layouts` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0599]: the associated item `STABLE_ABI` exists for struct `GetTypeLayoutCtor<u128>`, but its trait bounds were not satisfied
  --> src/node.rs:15:38
   |
15 | #[cfg_attr(feature = "c-abi", derive(abi_stable::StableAbi))]
   |                                      ^^^^^^^^^^^^^^^^^^^^^ associated item cannot be called on `GetTypeLayoutCtor<u128>` due to unsatisfied trait bounds
   |
   = note: the following trait bounds were not satisfied:
           `u128: StableAbi`
   = note: this error originates in the macro `$crate::_sabi_type_layouts` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0599`.
error: could not compile `value-trait` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -22,7 +22,6 @@ jobs:
           - ""
           - "--features c-abi"
           - "--features 128bit"
-          - "--features 128bit,c-abi"
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v1

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,7 +17,8 @@ ryu = "1"
 halfbrown = "0.1"
 float-cmp = "0.9"
 
-# NOTE: Temporary until the canges are upstreamed
+# FIXME: Temporary until the canges are upstreamed:
+# https://github.com/rodrimati1992/abi_stable_crates/pull/61
 abi_stable = { version = "0.10", optional = true, git = "https://github.com/marioortizmanero/abi_stable_crates.git", branch = "floating-point" }
 
 [features]

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -12,6 +12,17 @@
 #![allow(clippy::module_name_repetitions, clippy::inline_always)]
 #![deny(missing_docs)]
 
+#[cfg(all(feature = "128bit", feature = "c-abi"))]
+compile_error! (
+    "Combining the features `128bit` and `c-abi` is impossible because i128's \
+    ABI is unstable (see \
+    https://github.com/rust-lang/unsafe-code-guidelines/issues/119). Please \
+    use only one of them in order to compile this crate. If you don't know \
+    where this error is coming from, it's possible that you depend on \
+    value-trait twice indirectly, once with the `c-abi` feature, and once with \
+    the `128bit` feature, and that they have been merged by Cargo."
+);
+
 use std::borrow::{Borrow, Cow};
 use std::convert::TryInto;
 use std::fmt;

